store/sqlstore/team_store.go

```diff
@@ -682,13 +681,11 @@ func (s SqlTeamStore) GetActiveMemberCount(teamId string) store.StoreChannel {
 				AND TeamMembers.TeamId = :TeamId
 				AND TeamMembers.DeleteAt = 0
 				AND Users.DeleteAt = 0`, map[string]interface{}{"TeamId": teamId})
-		if err != nil {
-			result.Err = model.NewAppError("SqlTeamStore.GetActiveMemberCount", "store.sql_team.get_member_count.app_error", nil, "teamId="+teamId+" "+err.Error(), http.StatusInternalServerError)
-			return
-		}
+	if err != nil {
+		return 0, model.NewAppError("SqlTeamStore.GetActiveMemberCount", "store.sql_team.get_member_count.app_error", nil, "teamId="+teamId+" "+err.Error(), http.StatusInternalServerError)
```

Comment:
can you change error message store.sql_team.get_member_count.app_error --> store.sql_team.get_active_member_count.app_error, just to be consistent

Reply:
OK

Reply:
just these 2 minor changes from my end. rest all look great!

Reply:
Both changes are done @mounicapaladugu, but It seems you have problems with Jenkins, sometimes if fails with minor changes